public/css/main.css

```diff
@@ -405,24 +386,21 @@ body.darkBG .progress {
   border-radius: 3px;
   z-index: 1;
   transform-origin: top right;
-  transition: transform 0.5s cubic-bezier(0.77,0.2,0.05,1.0),
-              background 0.5s cubic-bezier(0.77,0.2,0.05,1.0),
-              width .3s,
-              opacity 0.55s ease;
+  transition: transform 0.5s cubic-bezier(0.77, 0.2, 0.05, 1.0), background 0.5s cubic-bezier(0.77, 0.2, 0.05, 1.0), width .3s, opacity 0.55s ease;
```

Comment:
Same legibility issue here.  I don't think it's worth the gain.  Perhaps I'm wrong though.

Reply:
You're not IMO. Will correct